server/selinux/server/pulp-celery.te

```diff
@@ -8,12 +8,19 @@ init_daemon_domain(celery_t, celery_exec_t)
 type pulp_var_cache_t;
 files_type(pulp_var_cache_t)
 
+type celery_var_run_t;
```

Comment:
I think `celery_var_run_t` should be called `pulp_var_run_t`. After looking at a system with SELinux enabled the convention seems to be to have the software itself (not its dependency) carry the context name. Even the `celery_t` context should probably be named `pulp_t` or `pulp_celery_t` but that is a [different issue](https://pulp.plan.io/issues/563).

This should be renamed to `pulp_var_run_t`.